sound/soc/sof/intel/hda.c

```diff
@@ -597,6 +597,8 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 	/* set default mailbox offset for FW ready message */
 	sdev->dsp_box.offset = HDA_DSP_MBOX_UPLINK_OFFSET;
 
+	snd_hda_set_power_save(sof_to_hbus(sdev), SND_SOF_SUSPEND_DELAY_MS);
```

Comment:
This can do with a comment explaining why this is being manually done here so anyone looking at the code undertsnds the flow with HDA codec.

Reply:
added comment